myapp/app/helpers/session_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    return unless session[:user_id]
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+    
+  def current_user?(user)
+    user == current_user
+  end
+    
+  def logged_in?
+    !current_user.nil?
```

Comment:
```suggestion
    current_user.present?
```
how about this? Does this work well?

Reply:
Modified this 👍